plugins/woocommerce/client/legacy/js/frontend/add-to-cart-variation.js

```diff
@@ -245,7 +245,7 @@
 			$dimensions    = form.$product.find(
 				'.product_dimensions, .woocommerce-product-attributes-item--dimensions .woocommerce-product-attributes-item__value'
 			),
-			$qty           = form.$singleVariationWrap.find( '.quantity' ),
+			$qty           = form.$singleVariationWrap.find( '.woocommerce-variation-add-to-cart .quantity' ),
```

Comment:
At the risk of overthinking this, what do you think of this alternative:

```suggestion
			$qty           = form.$singleVariationWrap.find( '.quantity input[name="quantity"]' ),
```

Or simply:

```suggestion
			$qty           = form.$singleVariationWrap.find( 'input[name="quantity"]' ),
```

My thinking is that this is (just slightly) more robust, since the input name is the key differentiator and since the surrounding div classes can be readily changed via template overrides.

I realize surrounding code still relies on CSS selectors, and that the input name can also be changed via a template override, but that change seems marginally less likely ... and any small tweak that might reduce the potential for breakages elsewhere is worth exploring.

Reply:
Hi @barryhughes is it possible to have multiple products on the same page? like with One Page Checkout or other builders? Would `name` alone be enough then as a selector? But otherwise, `.find( 'input[name="quantity"]' )` does resolve my problem.

Reply:
I guess that's possible, though we are still looking within the *single variation wrapper,* here. If it otherwise suits your needs, perhaps let's run with the first suggestion (`.quantity input[name="quantity"]`).

Reply:
True.. that probably means it should only happen once per `<form>` unless you're doing something super custom (and I'm already out on the edge of super custom with this project). Works for me! Thanks for reviewing this so quickly.